USECASES.md

```diff
@@ -1,117 +1,117 @@
-
-Ben Hutten notes some thoughts on the topic of matching in the [MME
-repository](https://github.com/ga4gh/mme-apis/wiki/Phenotype-matching).
-
-## Aggregation, Grouping and Sorting
-Responses were mixed on the topic of aggregation. Responses on the topic of
-"Aggregate functions (eg. minimum, maximum, average) range from "not needed"
-(1), "nice to have" (3) to "must have" (2). There is a division here around
-whether this sort of operation should be performed at query time or after the
-results are generated. One important point of consideration is that in some
-cases data may be available to a particular user only in aggregate meaning that
-it must be performed as part of the query.
-
-The closely related feature of grouping (for example, counting the number of
-variants grouped by gene) was rated as "not needed" (3), "nice to have" (1) and
-"must have" (2) by respondents.
-
-Sorting, or the related concept of ranking, was rated as either "not needed"
-(2) or "must have" (2) by respondents.
-
-## Data types
-The GA4GH Search API is not expected to define data types, but it will be important
-that some shared vocabulary are available to facilitate federated queries.
-
-All responses in the feature matrix indicated that the availability of a
-standard set of fields is "nice to have" or "must have", while the ability to
-have arbitrary non-standard fields is a "must have".
-
-Data type mentioned in the responses include:
-
-* Variant
-* Age
-* Sex
-* HPO terms
-* Contact information
-* Method of inheritance
-* Consent types (eg, to find patients with data consented for a particular use)
-* Various associated clinical data and/or assessments 
-
-In terms of how to represent these types in the API, there are a few
-suggestions:
-
-* Schema Blocks from GA4GH
-* schema.org from W3C and the related DCAT project
-* DATS (a joint collaboration funded by NIH)
-
-# Caveats
-
-## Small number of responses
-This summary was produced from a relatively small amount of response data, and
-the individual responses were often collected against an evolving document. In
-particular, the feature matrix doubled in size between some responses, and the
-questionnaire was changed completely between the two responses.
-
-## Lack of clarity in feature definitions
-The items expressed in the feature matrix are not fully defined, leaving their
-interpretation up to the reader. In some cases, respondents added comments to
-try to clarify how they interpreted the feature.
-
-# Open Questions
-If you have an answer to one of these questions, please either file an issue,
-open a PR against this document or reach out to the list at
-ga4gh-discovery-search@ga4gh.org.
-
-* How exactly are fuzzy matches for phenotypes being performed today? For
-  example: this term and up the tree, this term and down the tree, other well
-  defined functions or something completely custom?
+﻿# Introduction
+This document attempts to summarize Data Connect use cases collected from key driver
```

Comment:
Maybe just drop `Data Connect` and leave `summarize use cases`.  I wasn't using `search` originally as the name of the specification, but a classification of user behaviour.  Alternatively, I would say something like `summarize a variety of use cases`...

Reply:
Done, thank you!